napari/_vendor/qt_json_builder/qt_jsonschema_form/widgets.py

```diff
@@ -564,6 +590,18 @@ def state(self) -> int:
     def setDescription(self, description: str):
         self._description.setText(description)
 
+    def setMaximum(self, value: int):
+        try:
+            super().setMaximum(value)
```

Comment:
why?

Reply:
Not all widgets provide a setmax set min method so I figured it would be simpler to add and try to run it if it is there on the widgets